presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -177,7 +178,7 @@ public void testTableScan()
     public void testProject()
     {
         TableScanNode tableScan = tableScan("ts", "orderkey");
-        PlanNode project = project("project", tableScan, "string", new Cast(new SymbolReference("orderkey"), "STRING"));
+        PlanNode project = project("project", tableScan, "string", new Cast(new SymbolReference("orderkey"), "VARCHAR"));
```

Comment:
unrelated change?

Reply:
No, i had to fix that. The fragment created here was not passing the validation, as the type called `STRING` doesn't exist in Presto